test/runtests.jl

```diff
@@ -1,7 +1,29 @@
 using GIFImages
 using GIFImages.LibGif
 using Test
+using Downloads
+
+_wrap(name) = "https://github.com/ashwani-rathee/gif-sampleimages/blob/main/$(name)?raw=true"
+get_example(x) = Downloads.download(_wrap(x))
 
 @testset "GIFImages.jl" begin
-    # Write your tests here.
+    @test size(gif_decode(get_example("fire.gif")))[1] == 33
```

Comment:
hmmm, why would you test `sz[1] == 33` instead of `sz == (33, ...)`?

Reply:
Actually I wanted to know about the number of images in the gif here